docs/layers/tile-3d-layer.md

```diff
@@ -110,26 +131,45 @@ This value is only applied when [tile format](https://github.com/AnalyticalGraph
 
 - Default [`Tiles3DLoader`](https://loaders.gl/modules/3d-tiles/docs/api-reference/tile-3d-loader)
 
-A loader which is used to decode the fetched tiles. Either a [`Tiles3DLoader`](https://loaders.gl/modules/3d-tiles/docs/api-reference/tile-3d-loader) or `I3SLoader`.
+A loader which is used to decode the fetched tiles. Available options are [`CesiumIonLoader`,`Tiles3DLoader`](https://loaders.gl/modules/3d-tiles/docs/api-reference/tile-3d-loader), [`I3SLoader`](https://loaders.gl/modules/i3s/docs/api-reference/i3s-loader).
 
-##### `loadOptions` (Object, Optional)
+##### `tilesetOptions` (Object, Optional)
```

Comment:
The whole point of having a nested `loadOptions` object is to be able to supply options for multiple loaders/steps, i.e.

```js
{
  'cesium-ion': {...}
  tiles: {...}
}
```